tools/kql/pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[project]
+name = "detection-rules-kql"
```

Comment:
We can rename these if needed. Note: this doesn't impact the imports or how the package is stored in site-packages. In this case, it'll still be the `kql` package.